migrations/20220402211149-add-place-name-slug.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addColumn(
+        'places',
+        'name_slug',
+        {
+          type: Sequelize.STRING,
+          allowNull: true,
+        },
+        { transaction },
+      );
+
+      const [places] = await queryInterface.sequelize.query(`select * from "places"`, { transaction });
+
+      for (const place of places) {
```

Comment:
Could you make sure places is not undefined and then run the query? 
I am worried it could fail for the empty database.

It looks like the sasme case as in `20220324070441-pluralize-user-place-table`

Reply:
It won't be `undefined` but an empty array, so it will work. Case of `20220324070441-pluralize-user-place-table` was different because we were performing `bulkUpdate` and passing an empty array there.